src/validators/RequestSchema.js

```diff
@@ -275,6 +275,12 @@ const UpdateSettingsRequestSchema = Joi.object().keys({
   logo: Joi.string().allow(""),
 })
 
+const UpdateRepoPasswordRequestSchema = Joi.object().keys({
+  encryptedPassword: Joi.string(),
+  iv: Joi.string(),
+  enablePassword: Joi.boolean(),
+})
```

Comment:
is this all or nothing? ie, do we accept partial inputs like `encryptedPassword + iv` only

Reply:
No partial input - always specify whether we're enabling or disabling the password